src/nvim/options.lua

```diff
@@ -1248,13 +1248,6 @@ return {
       varname='p_inf',
       defaults={if_true=false}
     },
-    {
-      full_name='insertmode', abbreviation='im',
-      short_desc=N_("start the edit of a file in Insert mode"),
-      type='bool', scope={'global'},
-      varname='p_im',
-      defaults={if_true=false}
-    },
```

Comment:
This should be kept, but change `varname` to `p_force_off` . compare the entry for `'compatible'`